chip-testing/test/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../../tools/tsconfig.base.json",
+    "compilerOptions": {
+        "outDir": "../build/test",
+        "target": "es2018",
```

Comment:
@Apollon77 the `target` tsconfig option proliferates with this PR.  I think we should just change to ES2018 in tsconfig.base.json.

My reasoning:

1. We use a number of ES2017 and ES2018 features that I'm pretty sure can't be backfilled (e.g. regexp "s" flag is ES2018)
2. More generally I'm 99% positive any hypothetical ES5 runtime has 0% chance of running matter.js
3. It would make transpiled code and a few of the new tsconfigs more compact

WDYT?

Doesn't need to be this PR (but could be 😉) just mentioning here because of the redundancy.